macosx/FileRenameSheetController.xib

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="11762" systemVersion="16D32" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none">
```

Comment:
What's changing in this xib?

Reply:
Nothing really, just touched by Xcode, I think it converted XIB to autolayout.
Let's decline this PR in favor in ivar convertion.